docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -216,6 +216,27 @@ The application connection tab shows an example command that can be used to quer
 Requests sent under the displayed address will be proxied through an authenticated tunnel to the
 application.
 
+## Hardware key support
```

Comment:
I'd put it right before the Configuration section. It's an extra feature that Connect supports that requires specific hardware, so I think it should be below sections that are applicable to a bigger user group.

Configuration is more like a reference section and anything below is not feature-related, hence this ordering where Hardware key support would be just above Configuration.

Reply:
Makes sense, done.